client/api/InnerClient.cs

```diff
@@ -243,12 +280,12 @@ public bool BoolVariation(string key, Target target, bool defaultValue)
                 {
                     return evaluator.BoolVariation(key, target, defaultValue);
                 }
+                
                 catch (InvalidCacheStateException)
                 {
                     if (logger.IsEnabled(LogLevel.Error))
```

Comment:
same as above, remove the noise (and it's an incorrect copy/paste)

Reply:
Ah - thanks for catching this.